ktbridge/src/jvmTest/kotlin/app/cash/quickjs/ktbridge/KtBridgeTest.kt

```diff
@@ -38,7 +39,8 @@ class KtBridgeTest {
   @After fun tearDown() {
     quickjs.close()
   }
-
+  
+  @Ignore("needs some gradle changes so the KtBridge plugin can be applied")
```

Comment:
You can apply the plugin without changes by using
```gradle
dependencies {
  add(KotlinPluginKt.PLUGIN_CLASSPATH_CONFIGURATION_NAME, project(':ktbridge:plugin'))
}
```
or whatever the equivalent kts is

Reply:
Done in follow-up.